templates/adspot/handlers/buy.ts

```
@@ -0,0 +1,73 @@
+        return {
+            buttons: [
+                { label: 'Visit', action: 'link', target: storage.currentAd.visitLink || config.visitLink },
+                { label: 'ℹ️', action: 'post' }
+            ],
+            component: PageView(config, storage, body.fid.toString()),
+            handler: 'page',
+        }
+    }
+
+    if (body.buttonIndex === 2) {
+        const bidAmount = parseFloat(body.inputText)
```

Biome

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)